internal/services/mssql/validate/database_sku_name.go

```diff
@@ -21,7 +21,8 @@ const (
 	Stretch          = "(DS(1|2|3|4|5|6|10|12|15|20)00)"
 	BusinessCritical = "(BC_M_(8|10|12|14|16|18|20|24|32|64|128))"
 	Gen4             = "((GP|HS|BC)_Gen4_(1|2|3|4|5|6|7|8|9|10|16|24))"
-	Gen5             = "(GP|HS|BC)_Gen5_(2|4|6|8|10|12|14|16|18|20|24|32|40|80)"
+	Gen5             = "(GP|BC)_Gen5_(2|4|6|8|10|12|14|16|18|20|24|32|40|80|128)"
+	HsGen5           = "(HS)_Gen5_(4|6|8|10|12|14|16|18|20|24|32|40|80)"
```

Comment:
The reason the CI tests are failing is because it looks like you missed the SLO `vCore` value of `2` in your new `HsGen5` const field:

```suggestion
	HsGen5           = "(HS)_Gen5_(2|4|6|8|10|12|14|16|18|20|24|32|40|80)"
```

I would also suggest adding max value tests here(e.g., `./internal/services/mssql/validate/database_sku_name_test.go`) as well since for the `Gen5` SLO's these can vary depending on the compute sizes(`GP`, `BC` and `HS`).

```go
		{
			name:  "Valid GP Gen5 vCore Max",
			input: "GP_Gen5_128",
			valid: true,
		},
		{
			name:  "Valid BC Gen5 vCore Max",
			input: "BC_Gen5_128",
			valid: true,
		},
		{
			name:  "Valid HS Gen5 vCore Max",
			input: "HS_Gen5_80",
			valid: true,
		},
		{
			name:  "Invalid HS Gen5 vCore Max",
			input: "HS_Gen5_128",
			valid: false,
		},
```